sdks/python/apache_beam/examples/datastore_wordcount.py

```diff
@@ -61,53 +65,134 @@ def process(self, context):
     return words
 
 
+class CreateEntityFn(object):
+  """Create a Cloud Datastore entity from the given string."""
```

Comment:
Could you open an issue and fix the naming consistency issue in a follow up PR. All references to `Google Cloud Datastore` should be the same. If you want to refer it in a short way maybe single word/non-capitalized `datastore` could be used.